vmdb/app/models/vm_or_template.rb

```diff
@@ -1165,7 +1165,7 @@ def self.refresh_ems(vm_ids)
   end
 
   def refresh_ems
-    raise "no #{ui_lookup(:table => "ext_management_systems")} or credentials defined" unless self.ext_management_system && self.ext_management_system.authentication_valid?
+    raise "no #{ui_lookup(:table => "ext_management_systems")} or credentials defined" unless self.ext_management_system && self.ext_management_system.has_credentials?
```

Comment:
Same answer as my question above...I think we should check both (or does authentication_status_ok handle missing credentials as well?)